composer.json

```diff
@@ -65,6 +66,7 @@
     "scripts": {
         "ci:php:lint": "find *.php Classes Configuration Tests -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
         "ci:php:sniff": ".Build/vendor/bin/phpcs Classes Tests",
+        "ci:json:lint": ".Build/vendor/bin/jsonlint ./*.json",
```

Comment:
Doesn't this limit the linting to files in the current directory (i.e., no subdirectories)?